src/data/cel.rs

```diff
@@ -441,6 +442,22 @@ fn properties<'e>(exp: &'e CelExpression, all: &mut Vec<Vec<&'e str>>, path: &mu
     }
 }
 
+#[allow(dead_code)]
```

Comment:
I'd put this behing a feature flag or something... if only as `cfg[test]`... wdyt?

Reply:
Done